### PR TITLE
Fix and optimize image alt tags for better SEO

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
This will boost traffic and SEO (Search Engine Optimization) for Bisq.network.

It's extremely important to give images good alt attributes: alt text strengthens the message of website pages with search engine spiders and improves the accessibility of it.

### CODE REVIEW

**Comment by m52go** [comment]
@RiccardoMasutti these are good edits, but the `index.html` page is never actually rendered to the user, so these edits won't ever take effect. 

Hence my other PR #279 to get rid of that page altogether...review appreciated :)

As for this PR, the changes should be made in the other files that are actually rendered like `homepage_content.html`, `os_selector_options.html`, etc.

I know this isn't intuitive...refactoring is needed.

**Comment by RiccardoMasutti** [comment]
@m52go ready to merge

File: _includes/homepage_content.html [hunk 1]
```diff
@@ -145,7 +145,7 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
               </div>
               <div class="col-md-12 order-md-1 text-md-left homepage-hero">
-                <img class="img-fluid shadow-lg" alt="bisq" style="width: 100%; display: block;" src="/images/bisq_screen0.png">
+                <img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" style="width: 100%; display: block;" src="/images/bisq_screen0.svg">
```
Comment (m52go) [inline _includes/homepage_content.html]: Why are image paths changed? [SVGs were replaced in favor of PNGs](https://github.com/bisq-network/bisq-website/pull/304) a short while ago because of a strange issue on macOS.

**Review by m52go — CHANGES_REQUESTED** [review]
NACK - see comment.

File: _includes/homepage_content.html (hunk 1 above)
Comment (RiccardoMasutti) [reply _includes/homepage_content.html]: ty

**Comment by RiccardoMasutti** [comment]
@m52go ty for review. fixed

File: _includes/homepage_content.html [hunk 2]
```diff
@@ -30,8 +30,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
                               <div class="id-mac hidden">
                                   <div class="float-left">
-                                      <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g">
-                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w">
+                                      <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac">
+                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac">
```
Comment (m52go) [inline _includes/homepage_content.html]:
```suggestion
                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Mac">
```

Why was this changed?

File: _includes/homepage_content.html [hunk 3]
```diff
@@ -52,8 +52,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
                               <div class="id-rpm64 hidden">
                                   <div class="float-left">
-                                      <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g">
-                                      <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w">
+                                      <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Redhat/Fedora">
```
Comment (m52go) [inline _includes/homepage_content.html]:
```suggestion
                                      <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Red Hat/Fedora">
```

File: _includes/homepage_content.html [hunk 4]
```diff
@@ -52,8 +52,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
...
+                                      <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Redhat/Fedora">
```
Comment (m52go) [inline _includes/homepage_content.html]:
```suggestion
                                      <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Red Hat/Fedora">
```

File: _includes/homepage_content.html [hunk 5]
```diff
@@ -86,22 +86,22 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
...
+                          <img src="/images/icon-android-g.svg" class="os-icon os-icon-g" alt="Download Bisq Notification App for Android">
+                          <img src="/images/icon-android-w.svg" class="os-icon os-icon-w" alt="Download Bisq Notification App for Android">
                           {{ item.androidNotificationsApp }}
                       </a>
 
 
                       <a href="https://itunes.apple.com/us/app/bisq-mobile/id1424420411" target="_blank" rel="noopener" class="downloads-ios hidden btn btn-lg btn-outline-secondary btn-icon hero-btn col-sm-12 col-md-12 col-lg-3 px-0 shadow-btn" onclick="ga('send', 'event', 'Jumbotron actions', 'download', 'ios-notifications');">
-                          <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g">
-                          <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w">
+                          <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq Notification App for iOs">
```
Comment (m52go) [inline _includes/homepage_content.html]:
```suggestion
                          <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq Notification App for iOS">
```

File: _includes/homepage_content.html [hunk 6]
```diff
@@ -86,22 +86,22 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
...
+                          <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq Notification App for iOs">
```
Comment (m52go) [inline _includes/homepage_content.html]:
```suggestion
                          <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq Notification App for iOS">
```

File: _includes/homepage_content.html [hunk 7]
```diff
@@ -263,14 +263,14 @@ <h2 id="join-the-community">{{ item.hJoin }}</h2>
...
-          <li><img src="/images/community/twitter.svg" alt="twitter"> <a href="https://twitter.com/bisq_network">{{ item.liTwitter }}</a></li>
-          <li><img src="/images/community/youtube.svg" alt="youtube"> <a href="https://www.youtube.com/c/bisq-network">{{ item.liYoutube }}</a></li>
-          <li><img src="/images/community/forum.svg" alt="forum"> <a href="https://bisq.community">{{ item.liForum }}</a></li>
-          <li><img src="/images/community/mailinglist.svg" alt="mailinglist"> <a href="https://lists.bisq.network/listinfo/bisq-contrib">{{ item.liMailingList }}</a></li>
-          <li><img src="/images/community/irc.svg" alt="irc"> <a href="https://webchat.freenode.net/?channels=bisq">{{ item.liFreenodeIRC }}</a></li>
-          <li><img src="/images/community/reddit.svg" alt="reddit"> <a href="https://www.reddit.com/r/bisq">{{ item.liSubreddit }}</a></li>
+          <li><img src="/images/community/github.svg" alt="GitHub repository"> <a href="https://github.com/bisq-network">{{ item.liGithub }}</a></li>
+          <li><img src="/images/community/keybase.svg" alt="Keybase page"> <a href="https://keybase.io/team/bisq">{{ item.liKeybase }}</a></li>
```
Comment (m52go) [inline _includes/homepage_content.html]:
```suggestion
          <li><img src="/images/community/keybase.svg" alt="Keybase team"> <a href="https://keybase.io/team/bisq">{{ item.liKeybase }}</a></li>
```

File: _includes/homepage_content_tr.html [hunk 8]
```diff
@@ -30,8 +30,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
                               <div class="id-mac hidden">
                                   <div class="float-left">
-                                      <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g">
-                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w">
+                                      <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac">
+                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac">
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Mac">
```

File: _includes/homepage_content_tr.html [hunk 9]
```diff
@@ -52,8 +52,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
                               <div class="id-rpm64 hidden">
                                   <div class="float-left">
-                                      <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g">
-                                      <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w">
+                                      <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Redhat/Fedora">
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
                                      <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Red Hat/Fedora">
```

File: _includes/homepage_content_tr.html [hunk 10]
```diff
@@ -52,8 +52,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
...
+                                      <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Redhat/Fedora">
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
                                      <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Red Hat/Fedora">
```

File: _includes/homepage_content_tr.html [hunk 11]
```diff
@@ -86,22 +86,22 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
...
+                          <img src="/images/icon-android-g.svg" class="os-icon os-icon-g" alt="Download Bisq Notification App for Android">
+                          <img src="/images/icon-android-w.svg" class="os-icon os-icon-w" alt="Download Bisq Notification App for Android">
                           {{ item.androidNotificationsApp }}
                       </a>
 
 
                       <a href="https://itunes.apple.com/us/app/bisq-mobile/id1424420411" target="_blank" rel="noopener" class="downloads-ios hidden btn btn-lg btn-outline-secondary btn-icon hero-btn col-sm-12 col-md-12 col-lg-3 px-0 shadow-btn" onclick="ga('send', 'event', 'Jumbotron actions', 'download', 'ios-notifications');">
-                          <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g">
-                          <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w">
+                          <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq Notification App for iOs">
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
                          <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq Notification App for iOS">
```

File: _includes/homepage_content_tr.html [hunk 12]
```diff
@@ -86,22 +86,22 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
...
+                          <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq Notification App for iOs">
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
                          <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq Notification App for iOS">
```

File: _includes/homepage_content_tr.html [hunk 13]
```diff
@@ -145,7 +145,7 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
               </div>
               <div class="col-md-12 order-md-1 text-md-left homepage-hero">
-                <img class="img-fluid shadow-lg" alt="bisq" style="width: 100%; display: block;" src="/{{ page.lang }}/images/bisq_screen0.png">
+                <img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" style="width: 100%; display: block;" src="/images/bisq_screen0.svg">
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
                <img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" style="width: 100%; display: block;" src="/{{ page.lang }}/images/bisq_screen0.png">
```

Why was image source changed here?

File: _includes/homepage_content_tr.html [hunk 14]
```diff
@@ -240,10 +240,10 @@ <h2>{{ item.hGetStarted }}</h2>
       </ol>
     </div>
     <div class="col-12 col-md-7 offset-0 offset-md-1">
-      <div class="getting-started-img step-image step-image-1 d-block"><img class="img-fluid shadow-lg" alt="bisq" src="/{{ page.lang }}/images/bisq_screen0.png"></div>
-      <div class="getting-started-img step-image step-image-2 d-none" ><img class="img-fluid shadow-lg" alt="bisq" src="/{{ page.lang }}/images/bisq_screen1.png"></div>
-      <div class="getting-started-img step-image step-image-3 d-none" ><img class="img-fluid shadow-lg" alt="bisq" src="/{{ page.lang }}/images/bisq_screen0.png"></div>
-      <div class="getting-started-img step-image step-image-4 d-none" ><img class="img-fluid shadow-lg" alt="bisq" src="/{{ page.lang }}/images/bisq_screen1.png"></div>
+      <div class="getting-started-img step-image step-image-1 d-block"><img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" src="/images/bisq_screen0.svg"></div>
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
      <div class="getting-started-img step-image step-image-1 d-block"><img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" src="/{{ page.lang }}/images/bisq_screen0.png"></div>
```

File: _includes/homepage_content_tr.html [hunk 15]
```diff
@@ -240,10 +240,10 @@ <h2>{{ item.hGetStarted }}</h2>
...
+      <div class="getting-started-img step-image step-image-2 d-none" ><img class="img-fluid shadow-lg" alt="Open Trades view inside Bisq" src="/images/bisq_screen1.svg"></div>
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
      <div class="getting-started-img step-image step-image-2 d-none" ><img class="img-fluid shadow-lg" alt="Open Trades view inside Bisq" src="/{{ page.lang }}/images/bisq_screen1.png"></div>
```

File: _includes/homepage_content_tr.html [hunk 16]
```diff
@@ -240,10 +240,10 @@ <h2>{{ item.hGetStarted }}</h2>
...
+      <div class="getting-started-img step-image step-image-3 d-none" ><img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" src="/images/bisq_screen0.svg"></div>
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
      <div class="getting-started-img step-image step-image-3 d-none" ><img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" src="/{{ page.lang }}/images/bisq_screen0.png"></div>
```

File: _includes/homepage_content_tr.html [hunk 17]
```diff
@@ -240,10 +240,10 @@ <h2>{{ item.hGetStarted }}</h2>
...
+      <div class="getting-started-img step-image step-image-4 d-none" ><img class="img-fluid shadow-lg" alt="Open Trades view inside Bisq" src="/images/bisq_screen1.svg"></div>
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
      <div class="getting-started-img step-image step-image-4 d-none" ><img class="img-fluid shadow-lg" alt="Open Trades view inside Bisq" src="/{{ page.lang }}/images/bisq_screen1.png"></div>
```

File: _includes/homepage_content_tr.html [hunk 18]
```diff
@@ -263,14 +263,14 @@ <h2 id="join-the-community">{{ item.hJoin }}</h2>
...
-          <li><img src="/images/community/twitter.svg" alt="twitter"> <a href="https://twitter.com/bisq_network">{{ item.liTwitter }}</a></li>
-          <li><img src="/images/community/youtube.svg" alt="youtube"> <a href="https://www.youtube.com/c/bisq-network">{{ item.liYoutube }}</a></li>
-          <li><img src="/images/community/forum.svg" alt="forum"> <a href="https://bisq.community">{{ item.liForum }}</a></li>
-          <li><img src="/images/community/mailinglist.svg" alt="mailinglist"> <a href="https://lists.bisq.network/listinfo/bisq-contrib">{{ item.liMailingList }}</a></li>
-          <li><img src="/images/community/irc.svg" alt="irc"> <a href="https://webchat.freenode.net/?channels=bisq">{{ item.liFreenodeIRC }}</a></li>
-          <li><img src="/images/community/reddit.svg" alt="reddit"> <a href="https://www.reddit.com/r/bisq">{{ item.liSubreddit }}</a></li>
+          <li><img src="/images/community/github.svg" alt="GitHub repository"> <a href="https://github.com/bisq-network">{{ item.liGithub }}</a></li>
+          <li><img src="/images/community/keybase.svg" alt="Keybase page"> <a href="https://keybase.io/team/bisq">{{ item.liKeybase }}</a></li>
```
Comment (m52go) [inline _includes/homepage_content_tr.html]:
```suggestion
          <li><img src="/images/community/keybase.svg" alt="Keybase team"> <a href="https://keybase.io/team/bisq">{{ item.liKeybase }}</a></li>
```

File: _includes/os_selector_options.html [hunk 19]
```diff
@@ -16,58 +16,65 @@
 
 
 <div class="dropdown-item btn-icon">
-  <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" />
-  <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" /> Download for Mac
+  <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac" />
+  <img src="/images/icon-apple-w.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac" />
```
Comment (m52go) [inline _includes/os_selector_options.html]:
```suggestion
  <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Mac" />
```

File: _includes/os_selector_options.html [hunk 20]
```diff
@@ -16,58 +16,65 @@
...
   <div class="float-right">
     <a class="dl-deb64" href="https://bisq.network/downloads/v{{ site.client_version }}/Bisq-64bit-{{ site.client_version }}.deb">.deb</a>
   </div>
 </div>
 
 
 <div class="dropdown-item btn-icon">
-  <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" />
-  <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" /> Download for Red Hat/Fedora
+  <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Redhat/Fedora" />
```
Comment (m52go) [inline _includes/os_selector_options.html]:
```suggestion
  <img src="/images/icon-fedora-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Red Hat/Fedora" />
```

File: _includes/os_selector_options.html [hunk 21]
```diff
@@ -16,58 +16,65 @@
...
+  <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Redhat/Fedora" />
```
Comment (m52go) [inline _includes/os_selector_options.html]:
```suggestion
  <img src="/images/icon-fedora-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Red Hat/Fedora" /> 
```

**Review by m52go — CHANGES_REQUESTED** [review]
Some small nits such as "Redhat" --> "Red Hat".

But bigger issues are more concerning: a number of images were changed when they shouldn't have been changed. This PR wasn't supposed to change image links at all -- please keep PRs focused on the declared changes. I assume these were inadvertent on your part, but for me as reviewer, identifying and correcting such changes is unexpected and time-consuming. I'm still not sure I caught all of them. 

Please consider diffing your changes next time before submitting to verify nothing unintended has crept in...and of course, it's critical that you set up a testing environment if you haven't already (feel free to reach out if you're not familiar with what I'm talking about).

**Comment by m52go** [comment]
@RiccardoMasutti there was still a mistake. Testing in your local browser would easily help find these kinds of issues.

Screenshot from 2020-01-27 16-16-31

**Comment by m52go** [comment]
ACK
